### PR TITLE
[13.0][MIG] stock_report_quantity_by_location

### PR DESCRIPTION
**Pull request by kranokporn**
This module add an additional reporting is Inventory by location.
Edit view page by remove the view type.

cc. @kittiu Can you review please?

### CODE REVIEW

File: stock_report_quantity_by_location/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* Eficent Business and IT Consulting Services, S.L. (https://www.eficent.com)
+  * Jordi Ballester Alomar <jordi.ballester@eficent.com>
+  * Hector Villarreal <hector.villarreal@eficent.com>
+  * Kranokporn Th. <kranokpornt@ecosoft.co.th>
```
Comment (ecosoft-odoo) [inline stock_report_quantity_by_location/readme/CONTRIBUTORS.rst]:
```suggestion

* Ecosoft (http://ecosoft.co.th)
  * Kranokporn Th. <kranokpornt@ecosoft.co.th>
```

**Review by ghost — APPROVED** [review]

File: stock_report_quantity_by_location/wizards/stock_report_quantity_by_location.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        comodel_name="stock.location", string="Locations", required=True
+    )
+
+    def open(self):
+        self.ensure_one()
+        self._compute_stock_report_by_location()
+        action = {
+            "type": "ir.actions.act_window",
+            "view_mode": "pivot,tree",
+            "name": _("Stock Report by Location"),
```
Comment (MeetKD) [inline stock_report_quantity_by_location/wizards/stock_report_quantity_by_location.py:20]: by -> By

File: stock_report_quantity_by_location/wizards/stock_report_quantity_by_location.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            "context": {
+                "search_default_quantity_gt_zero": 1,
```
Comment (MeetKD) [inline stock_report_quantity_by_location/wizards/stock_report_quantity_by_location.py:22]: I didnt see the use of this key. Is this needed ?

**Review by MeetKD — CHANGES_REQUESTED** [review]

File: stock_report_quantity_by_location/wizards/stock_report_quantity_by_location.py (hunk 3 above)
Comment (bimajatiwijaya) [reply stock_report_quantity_by_location/wizards/stock_report_quantity_by_location.py:22]: It use to show only product quantity > 0.

File: stock_report_quantity_by_location/i18n/es_MX.po [hunk 4]
```diff
@@ -0,0 +1,161 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_report_quantity_by_location
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (bimajatiwijaya) [inline stock_report_quantity_by_location/i18n/es_MX.po:7]: Need to update 12.0 -> 13.0

File: stock_report_quantity_by_location/i18n/stock_report_quantity_by_location.pot [hunk 5]
```diff
@@ -0,0 +1,159 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_report_quantity_by_location
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (bimajatiwijaya) [inline stock_report_quantity_by_location/i18n/stock_report_quantity_by_location.pot:7]: Need to update 12.0 -> 13.0

**Review by bimajatiwijaya — CHANGES_REQUESTED** [review]

**Comment by JordiBForgeFlow** [comment]
Closing in favor of https://github.com/OCA/stock-logistics-reporting/pull/144

**Pull request closed without merging**